ui/app/models/secret-engine.js

```diff
@@ -160,6 +160,7 @@ export default class SecretEngineModel extends Model {
     return 'vault.cluster.secrets.backend.list-root';
   }
 
+  /* eslint-disable-next-line no-dupe-class-members */
   get accessor() {
     if (this.version === 2) {
       return `v2 ${this.accessor}`;
```

Comment:
@Monkeychip it looks like you added this getter which is overriding the `accessor` attr and triggering this lint rule. I wonder if we should simply change the name of the getter or move this logic to the serializer and transform the value when normalizing?

Reply:
I think it was for display purposes, I went ahead and added back the [concat on the template file](https://github.com/hashicorp/vault/pull/20481/files) in backends.hbs